tests/Contao/ValidatorTest.php

```diff
@@ -37,7 +37,7 @@ public function testValidatesEmailAddresses(string $email, $expected): void
     }
 
     /**
-     * @return (string,int)[]
+     * @return array(string,int)[]
```

Comment:
I think this should be `(string,int)[][]`?

Reply:
will change it to `(string|int)[][]`